analysers/analyser_merge_defibrillateurs_FR_basrhin.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Nicolas Bétheuil 2019                                      ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, GeoJSON, Load, Mapping, Select, Generate
+
+
+class Analyser_merge_defibrillateurs_FR_basrhin(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        self.missing_official = {"item":"8321", "class": 10, "level": 3, "tag": ["merge"], "desc": T_(u"Defibrillator not integrated") }
+
+        Analyser_Merge.__init__(self, config, logger,
+            u"https://www.data.gouv.fr/fr/datasets/donnee-thematique-localisation-des-defibrillateurs-automatiques-externes-dae-bas-rhin/",
+            u"DONNEE THEMATIQUE : Localisation des Défibrillateurs Automatiques Externes (DAE) - Bas-Rhin",
+            GeoJSON(Source(attribution = u"data.gouv.fr:Service Départemental d'Incendie et de Secours du Bas-Rhin", millesime = "23/04/2018",
+                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/40f84199-8b19-492d-9f61-45156ac73b63"),
+                    ),
+            Load("geom_x", "geom_y"),
+            Mapping(
+                select = Select(
+                    types = ["nodes"],
+                    tags = {"emergency": "defibrillator"}),
+                conflationDistance = 500,
+                generate = Generate(
+                    static1 = {
+                        "emergency": "defibrillator",
+                    },
+                    static2 = {"source": self.source},
+                text = lambda tags, fields: {"en": u"%s, %s, %s" % (fields["LOCALISATI"], fields["PRECISIONS"], fields["HORAIRES"])}
+                )
+            )
+        )
```

Comment:
I know it is not usual, but can you, follow the closing schema of other merge source files, please.

Reply:
Please inline more ) and }. It is for consistency.